2025-03-02 17:40:45,131 INFO: Locating template 'blog/includes/sidebar.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/blog/includes/sidebar.html')
None
2025-03-02 17:40:48,006 INFO: Locating template 'auth/login.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/auth/login.html')
None
2025-03-02 17:40:48,008 INFO: Locating template 'base.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/base.html')
None
2025-03-02 17:40:48,012 INFO: Locating template 'macros/csrf.html':
    1: trying loader of application 'app'
       class: jinja2.loaders.FileSystemLoader
       encoding: 'utf-8'
       followlinks: False
       searchpath:
         - /data/myblog/app/templates
       -> found ('/data/myblog/app/templates/macros/csrf.html')
None
2025-03-02 17:40:48,025 ERROR: 登录过程中发生错误: Could not build url for endpoint 'auth.register'. Did you mean 'auth.login' instead?
Traceback (most recent call last):
  File "/data/myblog/app/controllers/auth/__init__.py", line 102, in login
    return render_template('auth/login.html', form=form)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/flask/templating.py", line 150, in render_template
    return _render(app, template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/flask/templating.py", line 131, in _render
    rv = template.render(context)
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/data/myblog/venv/lib/python3.12/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/data/myblog/app/templates/auth/login.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/data/myblog/app/templates/base.html", line 43, in top-level template code
    {% block content %}{% endblock %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/app/templates/auth/login.html", line 34, in block 'content'
    <a href="{{ url_for('auth.register') }}">没有账号？立即注册</a>
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/flask/app.py", line 1074, in url_for
    return self.handle_url_build_error(error, endpoint, values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/flask/app.py", line 1063, in url_for
    rv = url_adapter.build(  # type: ignore[union-attr]
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/werkzeug/routing/map.py", line 919, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.exceptions.BuildError: Could not build url for endpoint 'auth.register'. Did you mean 'auth.login' instead?

None
2025-03-02 17:40:48,027 ERROR: Exception during request: Could not build url for endpoint 'auth.register'. Did you mean 'auth.login' instead?
None
2025-03-02 17:40:51,988 ERROR: 登录过程中发生错误: Could not build url for endpoint 'auth.register'. Did you mean 'auth.login' instead?
Traceback (most recent call last):
  File "/data/myblog/app/controllers/auth/__init__.py", line 102, in login
    return render_template('auth/login.html', form=form)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/flask/templating.py", line 150, in render_template
    return _render(app, template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/flask/templating.py", line 131, in _render
    rv = template.render(context)
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/data/myblog/venv/lib/python3.12/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/data/myblog/app/templates/auth/login.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/data/myblog/app/templates/base.html", line 43, in top-level template code
    {% block content %}{% endblock %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/app/templates/auth/login.html", line 34, in block 'content'
    <a href="{{ url_for('auth.register') }}">没有账号？立即注册</a>
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/flask/app.py", line 1074, in url_for
    return self.handle_url_build_error(error, endpoint, values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/flask/app.py", line 1063, in url_for
    rv = url_adapter.build(  # type: ignore[union-attr]
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/werkzeug/routing/map.py", line 919, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.exceptions.BuildError: Could not build url for endpoint 'auth.register'. Did you mean 'auth.login' instead?

None
2025-03-02 17:40:51,990 ERROR: Exception during request: Could not build url for endpoint 'auth.register'. Did you mean 'auth.login' instead?
None
2025-03-02 17:40:52,115 ERROR: 登录过程中发生错误: Could not build url for endpoint 'auth.register'. Did you mean 'auth.login' instead?
Traceback (most recent call last):
  File "/data/myblog/app/controllers/auth/__init__.py", line 102, in login
    return render_template('auth/login.html', form=form)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/flask/templating.py", line 150, in render_template
    return _render(app, template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/flask/templating.py", line 131, in _render
    rv = template.render(context)
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/data/myblog/venv/lib/python3.12/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/data/myblog/app/templates/auth/login.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/data/myblog/app/templates/base.html", line 43, in top-level template code
    {% block content %}{% endblock %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/app/templates/auth/login.html", line 34, in block 'content'
    <a href="{{ url_for('auth.register') }}">没有账号？立即注册</a>
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/flask/app.py", line 1074, in url_for
    return self.handle_url_build_error(error, endpoint, values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/flask/app.py", line 1063, in url_for
    rv = url_adapter.build(  # type: ignore[union-attr]
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/myblog/venv/lib/python3.12/site-packages/werkzeug/routing/map.py", line 919, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.exceptions.BuildError: Could not build url for endpoint 'auth.register'. Did you mean 'auth.login' instead?

None
2025-03-02 17:40:52,117 ERROR: Exception during request: Could not build url for endpoint 'auth.register'. Did you mean 'auth.login' instead?
None
